spec/models/post_spec.rb

```diff
@@ -0,0 +1,13 @@
+require "rails_helper"
+
+RSpec.describe Post, type: :model do
+  let(:blank_values) { [nil, ""] }
+  it { should have_valid(:body).when("Any 1 .!@#$%^&*}text") }
+  it { should_not have_valid(:body).when(*blank_values,
+       "This post is going to be longer than 255 characters 
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Trailing whitespace detected.